#! /bin/sh /usr/share/dpatch/dpatch-run
## 23-addd-sh-shebang.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: adds missing sh shebang in a script

@DPATCH@
--- ACE_wrappers.orig/bin/add_rel_link.sh
+++ ACE_wrappers/bin/add_rel_link.sh
@@ -1,3 +1,4 @@
+#!/bin/sh
 # This script assumes it is being run by bash
 
 # ======= needed functions ======
